components/dashboard/src/service/public-api.ts

```diff
@@ -241,7 +187,8 @@ function createServiceClient<T extends ServiceType>(
                 }
                 return (async function* () {
                     try {
-                        const client = await resolveClient();
+                        // for server streaming, we prefer jsonRPC
```

Comment:
for future reference: the only streaming methods we implement in our public api v1 are:
- `WorkspaceService.WatchWorkspaceStatus`
- `PrebuildService.WatchPrebuild`